specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/examples/NetworkWatcherMultiEndpointsConnectionMonitorCreate.json

```diff
@@ -0,0 +1,145 @@
+{
+  "parameters": {
+    "api-version": "2019-06-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "networkWatcherName": "nw1",
+    "connectionMonitorName": "cm1",
+    "location": "centraluseuap",
+    "parameters": {
+      "properties": {
+        "endpoints": [
+          {
+            "name": "vm1",
+            "resourceId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Compute/virtualMachines/DSCPTest-VM3"
+          },
+          {
+            "name": "vm2",
+            "resourceId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Compute/virtualMachines/DSCPTest-VM4"
+          }
+        ],
+        "tests": [
```

Comment:
this likely should be "testGroups" to match specs. the same in the responses below.

Reply:
yes, thx! fixed